subprojects/plugins/src/integTest/groovy/org/gradle/api/plugins/ProjectInternalViewDependencyIntegrationTest.groovy

```diff
@@ -17,10 +17,12 @@
 package org.gradle.api.plugins
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import spock.lang.Ignore
 
 /**
  * Tests that {@code JvmComponentDependencies#projectInternalView()} behaves as expected.
  */
+@Ignore("removed from 8.0")
```

Comment:
Why not just delete this class?

Reply:
I added this when I thought I was going to be ignoring more tests vs changing/deleting